tests/UserArgsTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GraphQL\Tests;
+
+use GraphQL\GraphQL;
+use GraphQL\Type\Definition\InputObjectType;
+use GraphQL\Type\Definition\ObjectType;
+use GraphQL\Type\Definition\Type;
+use GraphQL\Type\Schema;
+use PHPUnit\Framework\TestCase;
+
+class UserArgsTest extends TestCase
+{
+    /**
+     * @see it('disallows non-existent scalar field on input')
```

Comment:
Are those test cases from graphql-js?

Reply:
No, they aren't. They're just tests of behavior one expects. Is the `@see` wrong to use here? 

The first test, with a scalar, just covers behavior wasn't tested elsewhere.

The second test, with the array, resulted in an error:
```TypeError: Argument 1 passed to GraphQL\Type\Definition\Type::getNullableType() must be an instance of GraphQL\Type\Definition\Type, null given, called in /editflow/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 77```
With the code in this pull request, the second test passes.

Reply:
No, they aren't. They're just tests of behavior one expects. Is the `@see` wrong to use here? 

The first test, with a scalar, just covers behavior wasn't tested elsewhere.

The second test, with the array, resulted in an error:
```TypeError: Argument 1 passed to GraphQL\Type\Definition\Type::getNullableType() must be an instance of GraphQL\Type\Definition\Type, null given, called in /editflow/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 77```
With the code in this pull request, the second test passes.

Reply:
Content looks good, but as you noted the annotations should go.

Reply:
I removed the annotations and updated the comments and test names.